packages/features/bookings/lib/handleCancelBooking.ts

```diff
@@ -368,13 +368,16 @@ async function handler(req: CustomRequest) {
   await Promise.all(promises);
 
   const workflows = await getAllWorkflowsFromEventType(bookingToDelete.eventType, bookingToDelete.userId);
+  const parsedMetadata = bookingMetadataSchema.safeParse(bookingToDelete.metadata || {});
 
   await sendCancelledReminders({
     workflows,
     smsReminderNumber: bookingToDelete.smsReminderNumber,
     evt: {
       ...evt,
-      metadata: { videoCallUrl: bookingMetadataSchema.parse(bookingToDelete.metadata || {})?.videoCallUrl },
```

Comment:
Setting metadata to `{ videoCallUrl: undefined }` results in a ZodError when we try to parse it again using the `bookingMetadataSchema` in the other component